pkg/webhook/server_test.go

```diff
@@ -174,6 +174,34 @@ var _ = Describe("Webhook Server", func() {
 			Expect(handler.injectedField).To(BeTrue())
 		})
 	})
+
+	Context("when using an unmanaged webhook server", func() {
```

Comment:
nit: I'd just write this as `it should be able to serve in unmanaged mode` (no need for the context, IMO)